addons/portal/static/src/js/portal_chatter.js

```diff
@@ -2,6 +2,7 @@ odoo.define('portal.chatter', function (require) {
 'use strict';
 
 var core = require('web.core');
+var Dialog = require('web.Dialog');
```

Comment:
Dialog doesn't seem to be used in your diff?

Reply:
Bloup